### PR TITLE
Add CSS Box Shadows Generator

### PR DESCRIPTION
**Pull request by arbaouimehdi**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [x] you only add one (!) new helper per pull request.
- [x] you have checked if an open PR already exists.
- [x] the submitted website is focused on a single, development related issue.
- [x] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

File: helpers/css-box-shadows-generator.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "CSS Box Shadows Generator",
+  "desc": "CSS Box Shadow Generator enables shadow manipulation and control over all shadow properties. It provides instant updates to shadow codes for easy copying and individual shadow descriptions.",
```
Comment (stefanjudis) [inline helpers/css-box-shadows-generator.json:3]: Could you adjust the `desc` field.

From the contribution guidelines

> the desc field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

**Comment by stefanjudis** [comment]
Thanks so much for contributing. Doesn't seem to work here though. :)

image

File: helpers/css-box-shadows-generator.json (hunk 1 above)
Comment (arbaouimehdi) [reply helpers/css-box-shadows-generator.json:3]: Is this description better?

"Create, edit, and generate multiple custom CSS box shadows."

**Comment by arbaouimehdi** [comment]
> 

I conducted a demo to showcase that the tool is working correctly.

https://github.com/stefanjudis/tiny-helpers/assets/2392987/6ff8456f-ca38-4b8b-a782-ae303e126834

**Comment by stefanjudis** [comment]
Might be a glitch on my end. Once the `desc` field fits the format, I'm happy to merge. 

Thanks so much for contributing. 🙇‍♂️

File: helpers/css-box-shadows-generator.json (hunk 1 above)
Comment (arbaouimehdi) [reply helpers/css-box-shadows-generator.json]: Description Changed 👍

**Comment by stefanjudis** [comment]
Thanks so much for contributing. 💯

**Pull request merged**